screens/rating.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image} from 'react-native';
+import { Font } from 'expo';
+
+const window = Dimensions.get("window")
+
+export default class Login extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'Montserrat-Regular': require('/Users/nona/shopu/assets/fonts/Montserrat-Regular.otf'),
+      'Montserrat-Medium': require('/Users/nona/shopu/assets/fonts/Montserrat-Medium.otf'),
+      'Montserrat-SemiBold': require('/Users/nona/shopu/assets/fonts/Montserrat-SemiBold.otf'),
+    });
+    this.setState({ fontLoaded: true });
+  }
+  render() {
+    return (
+      <View>
+      {
+        this.state.fontLoaded ? (
+          <View style={styles.container}>
+            <View style={styles.topBanner} />
+            <Image style={styles.shoppingCarticon}
+                source={require('/Users/nona/shopu/images/shopping-cart.png')} />
+            <Image style={styles.personIcon}
+                source={require('/Users/nona/shopu/images/profile_person.png')} />
+            <Text style={styles.thankYouforText}>Thank U for</Text>
+            <Text style={styles.usingText}>using</Text>
+            <Text style={styles.shopuText}>shopU!</Text>
+            <View style={styles.ratingBox} />
+            <Image style={styles.ratingIcons}
```

Comment:
This can't just be images - this needs to be individual stars

Instead of having just an `Image` component, you should use the `View` component, and set its background to the star image.

Reply:
So how do I split the stars from that image? I saved the rating_group.png as a component from Figma. Do I need to make 20 Views?